lnbits/core/models.py

```diff
@@ -106,6 +106,8 @@ def from_row(cls, row: Row):
 
     @property
     def tag(self) -> Optional[str]:
+        if self.extra is None:
+            return ""
```

Comment:
What's happening here?

Reply:
self.extra, can be None, if so ```self.extra.get()``` would error.
so i just check it, and return a empty string if it is ```None``` instead of the tag.

Reply:
maybe, i can return None? because the return Value is Optional[str]